packages/wave/assets/css/site/player.css

```
@@ -56,6 +57,42 @@
 }
 
+.button-play.is-buffering svg,
+.post-play.is-buffering svg {
+    visibility: hidden;
+}
+
+.player-spinner {
+    display: none;
+    width: 22px;
+    height: 22px;
+    border: 2px solid currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)